src/entry-point-client/PageList.tsx

```diff
@@ -33,6 +34,12 @@ const PageList: PageInfo[] = [
     isPrivate: false,
     contents: <GetInvolvedPage />,
   },
+  {
```

Comment:
Since this component is now going to be used in a feature that's going into the _next_ release, we should make sure we remove this from page list after we've tested it and before we merge this PR.